<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kubuntu/Ubuntu 17.04+ Load SVG bug</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_path_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>871</width>
      <height>23</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_icon_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_path_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>871</width>
      <height>23</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_icon_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_size_1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>781</width>
      <height>23</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_size_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>781</width>
      <height>23</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_qt_ver">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>871</width>
      <height>23</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>qt ver</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_kernel_ver">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>871</width>
      <height>23</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>kernel ver</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_openImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>161</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string>Open image</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
